# /database/srv/GetPotInfo.srv

uint32 id
---
bool success
PotInfo info
